contexts/credentials/v2

```diff
@@ -112,9 +112,9 @@
       }
     },
 
-    "JsonSchema2023": {
+    "JsonSchema": {
```

Comment:
This looks like a duplicate of what's below. This one should be removed as the one below looks more correct (has `jsonSchema` defined in its scoped context).

Reply:
updated

Reply:
updated